src/sdam/monitor.ts

```diff
@@ -275,6 +299,8 @@ function checkServer(monitor: Monitor, callback: Callback<Document | null>) {
       hello.isWritablePrimary = hello[LEGACY_HELLO_COMMAND];
     }
 
+    // FIXME: Figure out how to set this. Should duration be the instantaneous rtt or the averaged
```

Comment:
This duration is linked to a particular heartbeat, so we should be using the instantaneous duration here.